testsupport/init.go

```diff
@@ -105,6 +108,74 @@ func waitForOperators(t *testing.T) {
 	t.Log("all operators are ready and in running state")
 }
 
+func getE2EServiceAccountToken(t *testing.T, hostNs string, apiConfigsa *api.Config, sacl client.Client) string {
+
+	// creating another config which is used for creating only resclient,
+	//so that the main kubeconfig is not altered
+	restkubeconfig, err := util.BuildKubernetesRESTConfig(*apiConfigsa)
+	require.NoError(t, err)
+
+	// Check if there is already a service account present for e2e test
+	sa := &corev1.ServiceAccount{}
+	err = sacl.Get(context.TODO(), types.NamespacedName{Namespace: hostNs, Name: "e2e-test"}, sa)
+
+	sacrb := &rbacv1.ClusterRoleBinding{}
+	err1 := sacl.Get(context.TODO(), types.NamespacedName{Namespace: hostNs, Name: "e2e-test"}, sacrb)
```

Comment:
the name of the ClusterRoleBinding is different
```suggestion
	err1 := sacl.Get(context.TODO(), types.NamespacedName{Namespace: hostNs, Name: "e2e-test-cluster-admin"}, sacrb)
```